### DISCUSSION TITLE
تغییر نام کانفیگ ها ؟

### DISCUSSION THREAD

**Discussion by BehnamAB** [hiddify/Hiddify-Manager#216] [Q&A سوالات رایج]
سلام میخواستم بدونم میشه کاری کرد که نام کانفیگها رو تغییر داد از داخل پنل که وقتی لینک ساب میدیم چیزی نشون داده شه که ما میخایم ؟ نه اینطوری 

tls_grpc_CDN_vmess_xxxxxxxxxxx

**Comment by zZedix** [comment]
این برای منم مهمه ! تغییر نام کانفیگها به صورت دلخواه خیلی مهمه ! یا حداقل اسم دامنه از انتهاش حذف بشه ! این کانفیگهارو بدی دست یه آدم ناتو براحتی تمام دامنه ها میوفته دستش

**Comment by MoeinMT** [comment]
منم همین مشکل رو دارم! چطوری میشه نام کانفیگ ها رو کامل تغییر داده؟
.منظورم قسمت اولیه نام کانفیگ نیست که میشه به راحتی توی تنظیمات پروکسی تغییر داد به صورت کامل اگه کسی بلده راهنمایی کنه

**Reply by zZedix** [reply]
متوجه نمیشم ؟ شما تو قسمت پروکسی میتونی کامل اسم کانفیگ رو تغییر بدی ! در قسمت دامنه هم نام نمایشی برای دامنه ! دیگه چیزی میمونه برای تغییر ؟ مثلا من اسم کانفیگ رو گذاشتم Vless و نام نمایش دامنه Reality وقتی میبرم توی کلاینت مینویسه 
Vless_Reality
